core/src/main/scala/dagr/core/execsystem/TaskExecutionInfo.scala

```diff
@@ -42,7 +42,6 @@ import dagr.core.tasksystem.Task
  * @param attemptIndex the one-based count of attempts to run this task.
  */
 class TaskExecutionInfo(var task: Task,
-                        val id: BigInt,
                         var status: TaskStatus.Value,
```

Comment:
Just wondering out loud ... whether `status`, `script` and `logFile` might all also become `private[core] var`s in Task in some future refactoring?